resources/js/LoginLink.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { router } from '@inertiajs/react'
+
+const LoginLink = ({
```

Comment:
For consistency with our other projects and packages, could you use a function component instead?

```tsx
export default function LoginLink
```